fallout/turfs/plating.dm

```diff
@@ -257,32 +257,51 @@
 /turf/open/floor/plating/ground/road
 	name = "\proper road"
 	desc = "A stretch of road."
-	icon = 'fallout/icons/turf/roadsidewalk.dmi'
-	icon_state = "road"
-	var/dir_variation = TRUE
+	icon = 'fallout/icons/turf/road.dmi'
+	icon_state = "road-255"
+	base_icon_state = "road"
+	smoothing_flags = SMOOTH_BITMASK
+	smoothing_groups = list(SMOOTH_GROUP_FALLOUT_ROAD)
+	canSmoothWith = list(SMOOTH_GROUP_FALLOUT_ROAD)
 
 /turf/open/floor/plating/ground/road/Initialize()
 	. = ..()
-	if(dir_variation)
-		dir = pick(GLOB.cardinals)
+	addtimer(CALLBACK(src, /atom/.proc/update_icon), 1)
 
-/turf/open/floor/plating/ground/road/curb
-	icon_state = "curb"
-	dir_variation = FALSE
+/turf/open/floor/plating/ground/road/update_icon()
+	var/crack_randomiser = "crack_[rand(1,18)]"
+	var/road_randomiser = "rand(-10,10)"
+	var/direction_randomiser = "rand(0,8)"
+	if(prob(20))
+		add_overlay(image('fallout/icons/turf/road.dmi', crack_randomiser, FLOAT_LAYER, direction_randomiser, road_randomiser, road_randomiser))
+		return
 
-/turf/open/floor/plating/ground/road/curb/corner
-	icon_state = "curbcorner"
-	dir_variation = FALSE
-
-/turf/open/floor/plating/ground/road/sidewalk
+/turf/open/floor/plating/ground/sidewalk
 	name = "sidewalk"
 	desc = "Paved tiles specifically designed for walking upon."
-	icon_state = "sidewalk"
-	dir_variation = FALSE
+	icon = 'fallout/icons/turf/sidewalk.dmi'
+	icon_state = "sidewalk-255"
+	base_icon_state = "sidewalk"
+	smoothing_flags = SMOOTH_BITMASK
```

Comment:
Why does this smooth, if all the icons are the same?

Reply:
Because it sets the overlay icons direction instead.

Reply:
Could you expand on this?
Bitmask smoothing doesn't change the `dir`, nor it touches overlays.
It just changes the `smoothing_junction` and updates the `icon_state` accordingly.

Reply:
on 

/turf/open/floor/plating/ground/sidewalk/update_icon()
	add_overlay(image('fallout/icons/turf/curb.dmi', icon_state, FLOAT_LAYER))
	if(prob(20))
		icon_state = "crack_[rand(1,11)]"
		dir = rand(0,8)

it uses the icon_state of the tile its placed on as an smoothable overlay of the same name.